index.js

```diff
@@ -0,0 +1,68 @@
+let opcaoSelectDiamante = document.getElementById('select-diamante');
+let divMostraDiamante = document.getElementById('mostra-diamante');
+let emBranco = '&nbsp'.repeat(3);
+let diamante = [];
+let meioDiamante;
+let texto = '';
+
+geraSelect();
+
+function mostrarDiamante(valor){
+    
+    for(let i=0;i<meioDiamante;i++)
+    {   
+        for(let j=0;j<valor;j++){
+            
+            if(meioDiamante-1 == j)
+            { 
+                let charCode = String.fromCharCode(i+65);
+                diamante[j-i] = charCode
+                diamante[j+i] = charCode
+            }           
+        }
+        preencheDiamante()    
+    }
+    
+    for(i= opcaoSelectDiamante.value -1;i>=1;i--)
+    { 
+        for(j=0;j<=valor;j++)
+            {
+                if(meioDiamante-1 == j)
+                { 
+                    diamante[(j-i)+1] = geraCharCode(i);
+                    diamante[(j+i)-1] = geraCharCode(i);
+                }
+            }
+            preencheDiamante()  
+    }
+    return divMostraDiamante.innerHTML=texto;  
+}
+
+function geraCharCode(indice){
+    let charCode = String.fromCharCode(64+indice);
+    return charCode;
```

Comment:
```suggestion
    return String.fromCharCode(64+indice);
```

Nesse caso, tu pode retornar diretamente.